src/game.h

```diff
@@ -1025,11 +1027,12 @@ class game
         void disp_kills();          // Display the player's kill counts
         void disp_faction_ends();   // Display the faction endings
         void disp_NPC_epilogues();  // Display NPC endings
-        void disp_NPCs();           // Currently UNUSED.  Lists global NPCs.
+    public:
+        void disp_NPCs();           // Currently for debug use.  Lists global NPCs.
+    private:
```

Comment:
Please rearrange the functions instead of introducing even more sections. I'd also consider making a `debug_menu` class and referencing it as a `friend` of the `game` class. This would prevent exposing too much of its guts.